sklearn/datasets/olivetti_faces.py

```diff
@@ -69,24 +69,27 @@ def fetch_olivetti_faces(data_home=None, shuffle=False, random_state=0,
         If False, raise a IOError if the data is not locally available
         instead of trying to download the data from the source site.
 
-    Returns
-    -------
-    An object with the following attributes:
-
-    data : numpy array of shape (400, 4096)
-        Each row corresponds to a ravelled face image of original size
-        64 x 64 pixels.
-
-    images : numpy array of shape (400, 64, 64)
-        Each row is a face image corresponding to one of the 40 subjects
-        of the dataset.
+    return_X_y : boolean, default=False.
+        If True, returns `(data, target)` instead of a `Bunch` object. See
+        below for more information about the `data` and `target` object.
 
-    target : numpy array of shape (400, )
-        Labels associated to each face image. Those labels are ranging from
-        0-39 and correspond to the Subject IDs.
+        .. versionadded:: 0.22
 
-    DESCR : string
-        Description of the modified Olivetti Faces Dataset.
+    Returns
+    -------
+    bunch : Bunch object with the following attributes:
+        - bunch.data: ndarray, shape (400, 4096). Each row corresponds to a
```

Comment:
Since it is stated these are attributes, this can be simplified:

```suggestion
        - data: ndarray, shape (400, 4096). Each row corresponds to a
```